test/remote/gateways/remote_orbital_test.rb

```diff
@@ -389,6 +490,34 @@ def test_successful_purchase_with_overridden_normalized_stored_credentials
     assert_equal 'Approved', response.message
   end
 
+  # Successful force capture on Electronic Check
+  def test_successful_force_capture_with_echeck
+    @options[:force_capture] = true
+    assert response = @gateway.purchase(@amount, @echeck, @options)
+    assert_success response
+    assert_equal 'APPROVAL        ', response.message
```

Comment:
Do you need all of this blank space after APPROVAL?

Reply:
Blank spaces are already produced in our response, that's why it was put here to match it exactly. However, we can the condition so that it checks whether APPROVAL is contained in the response.message.